ci/check-nix-tools.nix

```diff
@@ -14,7 +14,10 @@ writeScript "check-nix-tools.sh" ''
 
   export PATH="${makeBinPath [ stdenv.shellPackage coreutils nixStable git gawk ]}:$PATH"
 
-  cd $(git rev-parse --show-toplevel)
+  if [ -z "''${BUILDKITE:-}" ]; then
+    # Go to top of project repo, unless running under CI.
```

Comment:
can you document why not under CI?

Reply:
I tried, it's a bit hard to explain, but the reason is basically to make the skeleton build work.